<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>K-Radar Drivable Area  by AVE Lab</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/Icon.ico</normaloff>icons/Icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="48,12">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,12">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="Line" name="line_2">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QToolButton" name="toggle_line">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Mark line</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/line.png</normaloff>icons/line.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toggle_rect">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Mark rectangular surface</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/rectangle.png</normaloff>icons/rectangle.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toggle_move">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Move Image</string>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/move.png</normaloff>icons/move.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="save_im">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string notr="true" extracomment="Save Image">Save Image</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/download.png</normaloff>icons/download.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="prev_im">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Load Previous Image</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/arrow-left.png</normaloff>icons/arrow-left.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="next_im">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Load Next Image</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/arrow-right.png</normaloff>icons/arrow-right.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_5">
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoom_plus">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/zoom-in.png</normaloff>icons/zoom-in.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="reset_zoom">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Fit Image to Canvas</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/enlarge2.png</normaloff>icons/enlarge2.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoom_minus">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/zoom-out.png</normaloff>icons/zoom-out.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_6">
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="undo">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Undo</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/undo2.png</normaloff>icons/undo2.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="clear_all">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Clear All</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/loop2.png</normaloff>icons/loop2.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="redo">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Redo</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/redo2.png</normaloff>icons/redo2.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="qlabel_image">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:white
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>

          <item>
           <widget class="QLabel" name="drivable_area_qlabel_image">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:white
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>

         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,3,48">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="open_folder">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: black; /* Blue */
padding:10px;
color:white;</string>
          </property>
          <property name="text">
           <string>Open Folder</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>       
        <item>
          <widget class="QPushButton" name="add_attention_points">
            <property name="font">
            <font>
              <pointsize>9</pointsize>
            </font>
            </property>
            <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
            </property>
            <property name="autoFillBackground">
            <bool>false</bool>
            </property>
            <property name="styleSheet">
            <string notr="true">background-color: orange; /* Blue */
  padding:10px;
  color:white;</string>
            </property>
            <property name="text">
            <string>Add Attention Points</string>
            </property>
            <property name="autoDefault">
            <bool>false</bool>
            </property>
            <property name="default">
            <bool>false</bool>
            </property>
            <property name="flat">
            <bool>false</bool>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QPushButton" name="generate_DA_label">
            <property name="font">
            <font>
              <pointsize>9</pointsize>
            </font>
            </property>
            <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
            </property>
            <property name="autoFillBackground">
            <bool>false</bool>
            </property>
            <property name="styleSheet">
            <string notr="true">background-color: blue; /* Blue */
  padding:10px;
  color:white;</string>
            </property>
            <property name="text">
            <string>Generate DA Label</string>
            </property>
            <property name="autoDefault">
            <bool>false</bool>
            </property>
            <property name="default">
            <bool>false</bool>
            </property>
            <property name="flat">
            <bool>false</bool>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QPushButton" name="save_Drivable_Area_label">
            <property name="font">
            <font>
              <pointsize>9</pointsize>
            </font>
            </property>
            <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
            </property>
            <property name="autoFillBackground">
            <bool>false</bool>
            </property>
            <property name="styleSheet">
            <string notr="true">background-color: red; /* Blue */
  padding:10px;
  color:white;</string>
            </property>
            <property name="text">
            <string>Save DA Label</string>
            </property>
            <property name="autoDefault">
            <bool>false</bool>
            </property>
            <property name="default">
            <bool>false</bool>
            </property>
            <property name="flat">
            <bool>false</bool>
            </property>
          </widget>
        </item>
<!--\
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>List of Images</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
-->
        <item>
          <widget class="QListWidget" name="list_of_attention_point_column">
                    <property name="styleSheet">
                    <string notr="true">background-color:yellow</string>
                    </property>
                    <property name="frameShape">
                    <enum>QFrame::Box</enum>
                    </property>
                    <property name="frameShadow">
                    <enum>QFrame::Plain</enum>
                    </property>
                    <property name="layoutMode">
                    <enum>QListView::Batched</enum>
                    </property>
                    <property name="batchSize">
                    <number>5</number>
                    </property>
          </widget>
        </item>

        <item>
          <widget class="QListWidget" name="list_of_attention_points_non_drivable_area_column">
                    <property name="styleSheet">
                    <string notr="true">background-color:orange</string>
                    </property>
                    <property name="frameShape">
                    <enum>QFrame::Box</enum>
                    </property>
                    <property name="frameShadow">
                    <enum>QFrame::Plain</enum>
                    </property>
                    <property name="layoutMode">
                    <enum>QListView::Batched</enum>
                    </property>
                    <property name="batchSize">
                    <number>5</number>
                    </property>
          </widget>
        </item>

        <item>
          <widget class="QPushButton" name="clear_all_attention_points_button">
            <property name="font">
            <font>
              <pointsize>9</pointsize>
            </font>
            </property>
            <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
            </property>
            <property name="autoFillBackground">
            <bool>false</bool>
            </property>
            <property name="styleSheet">
            <string notr="true">background-color: gray; /* Blue */
  padding:10px;
  color:white;</string>
            </property>
            <property name="text">
            <string>Clear All Attention Points</string>
            </property>
            <property name="autoDefault">
            <bool>false</bool>
            </property>
            <property name="default">
            <bool>false</bool>
            </property>
            <property name="flat">
            <bool>false</bool>
            </property>
          </widget>
        </item>

        <item>
         <widget class="QListWidget" name="qlist_images">
          <property name="styleSheet">
           <string notr="true">background-color:white</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::Batched</enum>
          </property>
          <property name="batchSize">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
